README.md

```diff
@@ -113,6 +113,85 @@ fastify.get('/', (req, reply) => {
 })
 ```
 
+#### `reply.cacheControl`
+The `reply` interface is decorated an helper to configure cache control response headers.
+```js
+// configure a single type
+fastify.get('/', (req, reply) => {
+  reply.cacheControl('public')
+  reply.send('ok')
+})
+
+// configure multiple types
+fastify.get('/', (req, reply) => {
+  reply.cacheControl('public')
+  reply.cacheControl('immutable')
+  reply.send('ok')
+})
+
+// configure a type time
+fastify.get('/', (req, reply) => {
+  reply.cacheControl('max-age', 42)
+  reply.send('ok')
+})
+
+// the time can be defined as string
+fastify.get('/', (req, reply) => {
+  reply.cacheControl('max-age', '1d') // will set to 'max-age=86400'
```

Comment:
Would it be interesting to mention the `ms` supported time formats in here?
https://github.com/vercel/ms